tests/conftest.py

```diff
@@ -43,6 +43,11 @@
     from collections.abc import Iterator
 
 
+@pytest.fixture(autouse=True)
+def debug_mode(monkeypatch):
+    monkeypatch.setenv("CRAFT_DEBUG", "1")
```

Comment:
Several tests (test_special_inputs, test_project_managed, etc.) are also setting the env var. I suppose they could be fixed, see:
- https://github.com/canonical/craft-application/blob/tests/integration/test_application.py#L273
- https://github.com/canonical/craft-application/blob/main/tests/integration/test_application.py#L180

Reply:
Thank you! I believe I've caught all of them now.